src/test/java/net/minecraftforge/debug/world/item/LazyCapabilitiesOnItemsTest.java

```diff
@@ -0,0 +1,165 @@
+package net.minecraftforge.debug.world.item;
```

Comment:
License header is missing

Reply:
What was the way to add those again?
checkLincenses?

Reply:
IIRC, the `licenseFormat` task.

Reply:
updateLicenses